src/Tasks/dependency-check-build-task/dependency-check-build-task.ts

```diff
@@ -26,6 +26,7 @@ async function run() {
         let failOnCVSS: string | undefined = tl.getInput('failOnCVSS');
         let suppressionPath: string | undefined = tl.getPathInput('suppressionPath');
         let reportsDirectory: string | undefined = tl.getPathInput('reportsDirectory');
+        let warnOnCVSSViolation: boolean | undefined = tl.getBoolInput('warnOnCVSSViolation', true);
```

Comment:
Will tell us if the "Only warn for found violations" option was set (true) or not (false)